sdks/java/core/build.gradle

```diff
@@ -95,6 +95,7 @@ dependencies {
   provided library.java.hamcrest
   provided 'io.airlift:aircompressor:0.18'
   provided 'com.facebook.presto.hadoop:hadoop-apache2:3.2.0-1'
+  provided library.java.zstd_jni
```

Comment:
There might be objections to adding zstd_jni as a dependency to core, but this is `provided` which I believe means optional.
cc: @kileys @lukecwik

Reply:
Provided is fine.